test/verify/check-storage-mounting

```diff
@@ -510,21 +381,150 @@ ExecStart=/usr/bin/sleep infinity
         mount("local", "nofail")
         self.content_tab_wait_in_info(1, 1, "Mount point", "ignore failure")
         self.assertIn("nofail", m.execute("findmnt --fstab -n -o OPTIONS /foo"))
-        self.assert_in_configuration(dev, "crypttab", "options", "nofail")
+        self.assert_in_configuration(disk, "crypttab", "options", "nofail")
         unmount()
 
         mount("nofail", "netdev")
         self.content_tab_wait_in_info(1, 1, "Mount point", "after network")
         self.assertIn("_netdev", m.execute("findmnt --fstab -n -o OPTIONS /foo"))
-        self.assert_in_configuration(dev, "crypttab", "options", "_netdev")
+        self.assert_in_configuration(disk, "crypttab", "options", "_netdev")
         unmount()
 
         mount("netdev", "never")
         self.content_tab_wait_in_info(1, 1, "Mount point", "never mount")
         self.assertIn("x-cockpit-never-auto", m.execute("findmnt --fstab -n -o OPTIONS /foo"))
-        self.assert_in_configuration(dev, "crypttab", "options", "noauto")
+        self.assert_in_configuration(disk, "crypttab", "options", "noauto")
         unmount()
 
 
+@testlib.nondestructive
```

Comment:
How is it nond when it does provisioning?

Reply:
Ugh, that's a copy pasta. Thanks for spotting! Fixed.